configure

```diff
@@ -217,12 +217,25 @@ install_local_rebar() {
     fi
 }
 
+install_local_emilio() {
+    if [ ! -x "${rootdir}/bin/emilio" ]; then
+        if [ ! -d "${rootdir}/src/emilio" ]; then
+            git clone --depth 1 https://github.com/cloudant-labs/emilio ${rootdir}/src/emilio
+        fi
+        cwd=`pwd`
+        cd ${rootdir}/src/emilio && ${REBAR} compile escriptize; cd ${cwd}
+        mv ${rootdir}/src/emilio/emilio ${rootdir}/bin/emilio
+        chmod +x ${rootdir}/bin/emilio
```

Comment:
Should also do a `cd ${rootdir}/src/emilio && ${REBAR} clean` at this point, like we do on L216 for `rebar`.